frontend/packages/vsphere-plugin/locales/en/vsphere-plugin.json

```
@@ -0,0 +1,54 @@
+  "Missing the vSphere config map.": "Missing the vSphere config map.",
+  "Not configured yet": "Not configured yet",
+  "Prometheus query failed.": "Prometheus query failed.",
+  "Invalid credentials": "Invalid credentials",
+  "Failing {{reason}}": "Failing {{reason}}",
+  "Failed to patch {{secret}}": "Failed to patch {{secret}}",
+  "Failed to create {{secret}} secret": "Failed to create {{secret}} secret",
+  "Failed to load kubecontrollermanager": "Failed to load kubecontrollermanager",
+  "Failed to patch kubecontrollermanager": "Failed to patch kubecontrollermanager",
+  "Failed to patch {{cm}}": "Failed to patch {{cm}}",
+  "Failed to create {{cm}} ConfigMap": "Failed to create {{cm}} ConfigMap",
+  "vsphere-connection-form-folderhelp-one": "Provide <1>datacenter</1> folder which contains VMs of the cluster, example: /<3>{{datacenter}}</3>/<5>vm</5>/<7>[MY_VMS_TOP_FOLDER]</7>.",
+  "vsphere-connection-form-datacenterhelp-one": "The name of an existing datacenter in the vSphere which the virtual machines backing this cluster are in.",
+  "vsphere-connection-form-datacenterhelp-two": "Please mind, that changing this value will break existing PersistentVolumes, if there are already any.",
+  "vsphere-connection-form-datastorehelp-one": "The name of an existing datastore in the datacenter where the persistent volumes will be stored.",
+  "vsphere-connection-form-datastorehelp-two": "Please mind, that changing this value will break existing PersistentVolumes, if there are already any.",
+  "vCenter": "vCenter",
+  "Enter the network address the vCenter is running on. It can either be a domain name or IP. If you're unsure, you can try to determine the value from the vSphere Web Client address. Example: ": "Enter the network address the vCenter is running on. It can either be a domain name or IP. If you're unsure, you can try to determine the value from the vSphere Web Client address. Example: ",
+  "Can be both domain name or IP, see additional info how to get it.": "Can be both domain name or IP, see additional info how to get it.",
+  "Username": "Username",
+  "Password": "Password",
+  "The password will be stored in a Secret in the kube-system namespace of this cluster.": "The password will be stored in a Secret in the kube-system namespace of this cluster.",
+  "Datacenter": "Datacenter",
+  "Default data store": "Default data store",
+  "Virtual Machine Folder": "Virtual Machine Folder",
+  "vSphere Problem Detector (can be outdated)": "vSphere Problem Detector (can be outdated)",
+  "Retry": "Retry",
+  "Delayed propagation of configuration": "Delayed propagation of configuration",
+  "After saving the configuration, it may take approximately one hour to see if the settings are correct and the operators' statuses are updated, nodes will get rebooted.": "After saving the configuration, it may take approximately one hour to see if the settings are correct and the operators' statuses are updated, nodes will get rebooted.",
```

A style suggestion to match documentation. See "May" in IBM style https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m 

```suggestion
  "After saving the configuration, it might take approximately one hour to see if the settings are correct and the operators' statuses are updated, nodes will get rebooted.": "After saving the configuration, it might take approximately one hour to see if the settings are correct and the operators' statuses are updated, nodes will get rebooted.",
```